js/bw.js

```diff
@@ -292,6 +292,8 @@ module.exports = class bw extends Exchange {
                 'info': currency,
                 'name': code,
                 'active': active,
+                'deposit': undefined,
+                'withdraw': undefined,
```

Comment:
I think `drawFee`, `dailyDrawLimit` and `drawFlag` are all about withdrawing, we should most likely use the drawFlag to determine the state of `withdraw` here.